<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PLMSettings</class>
 <widget class="QWidget" name="PLMSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QLabel" name="label_CloPlugin_Key">
       <property name="text">
        <string>CLO-Vise Version:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label_PlmSchema_Value">
       <property name="text">
        <string>Not Available</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_Cloplugin_Value">
       <property name="text">
        <string>Not Available</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="lable_PlmVersion_Value">
       <property name="text">
        <string>Not Available</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="label_PlmServer_Value">
       <property name="text">
        <string>Not Available</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_BackendConsumer_Key">
       <property name="text">
        <string>Backend Consumer Key:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_PLMTenant_Key">
       <property name="text">
        <string>PLM Tenant:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_PlmServer_Key">
       <property name="text">
        <string>PLM Server URL:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_ServerUserName_Key">
       <property name="text">
        <string>Server User Name:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLabel" name="label_ServerUser_Value">
       <property name="text">
        <string>Not Available</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_ConnectionStatus_key">
       <property name="text">
        <string>Connection status:</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLabel" name="label_ConnectionStatus_Value">
       <property name="text">
        <string>Not Available</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="label_WebAppConsumerKey_Value">
       <property name="text">
        <string>Not Available</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_WebAppConsumer_Key">
       <property name="text">
        <string>Web App Consumer Key:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="comboBox_PLMTenant">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>20</height>
        </size>
       </property>
       <property name="duplicatesEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="label_ConsumerKey_Value">
       <property name="text">
        <string>Not Available</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_PlmSchema_Key">
       <property name="text">
        <string>Active PLM Schema:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lable_PLMVersion_Key">
       <property name="text">
        <string>PLM Client Version:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_CompanyName_Key">
       <property name="text">
        <string>Company Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_CompanyName_Value">
       <property name="text">
        <string>Not Available</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
